### ISSUE TITLE
Move `$queryRaw()` to `$dangerouslyQueryRaw()`

### ISSUE THREAD

**Issue by cloudagon** [prisma/prisma#7095]
## Problem
It can be confusing for newer developers that these two are not the same:
```js
// value is escaped
queryRaw`SELECT * FROM table WHERE col=${value}` 

// open to injection
queryRaw(`SELECT * FROM table WHERE col=${value}`)
```

The same is true for `$executeRaw`. This is quite a dangerous mistake to make and so these functions should not be so close together, under the same name.

## Suggested solution

Keep the tagged template literal and move `$queryRaw()` and `executeRaw()` to `$dangerouslyQueryRaw()` and `$dangerouslyExecuteRaw()` (as React has done with `dangerouslySetInnerHTML`).

## Alternatives

Another thing worth considering is to remove the tagged template literal `queryRaw` because the "raw" already implies it is unsanitized. The sanitized tagged template literal could perhaps be moved under `sql` instead. The truly raw one could stay under `queryRaw()` or move to `$dangerouslyQueryRaw()`.

**Comment by ae42faf7** [comment]
Holy :hankey:, I had no idea `queryRaw()` was unsafe.
I like the suggested solution.

**Comment by Karl-EdwardFPJeanMehu** [comment]
I agree with @cloudagon, However, renaming $queryRaw to $sql or sql can also cause some confusion considering that all the RDBMSs currently supported or will be supported has some variant of SQL. I believe simply rename it to $query is the best choice, especially considering that MongoDB support will hopefully be available and it does not use SQL whatsoever.

**Comment by pantharshit00** [comment]
Looks like @matthewmueller opened a general issue on this topic so I referenced this there: https://github.com/prisma/prisma/issues/7142
